plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -380,4 +380,61 @@ private void assertOneNotNullResult(String query)
         assertEquals(results.getMaterializedRows().get(0).getFieldCount(), 1);
         assertNotNull(results.getMaterializedRows().get(0).getField(0));
     }
+
+    /*
+     * Content Merge from BaseMongoDistributedQueries.java
+     * supportsDelete() supportsViews() supportsCommentOnTable() supportsCommentOnColumn() not include
```

Comment:
This comment explains what you do, but not what the code is.
Commit message would be a better place, but in this case you can just remove it.